src/app/views/landlord/new-issue/new-issue.page.html

```diff
@@ -0,0 +1,27 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-button slot="start" shape="round" size="medium" (click)="cancelPressed()">
+      <ion-icon name="close" ></ion-icon>
+    </ion-button>
+    <ion-title>Create New Issue</ion-title>
+    <ion-button slot="end" shape="round" size="medium" (click)="checkPressed()">
+      <ion-icon name="checkmark">
+      </ion-icon></ion-button>
```

Comment:
Any thoughts on these closing tags @Gubbsy  @tobybessant ? I'd drop the second down

Reply:
Agreed